docs/monitoring.md

```diff
@@ -15,6 +15,12 @@ The repo includes an [example helm chart](https://github.com/IBM/varnish-operato
 After you have the [operator installed](installation.md), clone the repo and install the helm chart.
 
 ```bash
+$ #install operator with monitoring enabled
+$ helm install varnish-operator varnish-operator/varnish-operator --wait \
+--set monitoring.grafanaDashboard.enabled=true \
+--set monitoring.grafanaDashboard.datasourceName="Prometheus-varnish-operator" \
```

Comment:
Here you set it to `Prometheus-varnish-operator`, is it just an arbitrary name?

Reply:
it comes from [here](https://github.com/IBM/varnish-operator/blob/main/config/samples/helm-charts/varnish-operator-monitoring/templates/grafana-prometheus-datasource.yaml#L13). Since it's a doc about that sample, the values should match so the user can simply copy paste and run those commands.